csharp_dotnetcore/16.Handling-Atachments/Handling_Attachments/Handling_Attachments.csproj

```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
```

Comment:
You might need the CodeAnalysis tags in here to get the Analyzers to run. Check with @daveta to get them.